docs/porting-guide.rst

```diff
@@ -8,21 +8,21 @@ While in CPython one always uses ``PyObject *`` to reference to Python objects,
 in HPy there are several types of handles that should be used depending on the
 life-time of the handle: ``HPy``, ``HPyField``, and ``HPyGlobal``.
 
-- `HPy` represents short lived handles that live no longer than the duration of
+- ``HPy`` represents short lived handles that live no longer than the duration of
   one call from Python to HPy extension function. Rule of thumb: use for local
   variables, arguments, and return values.
 
-- `HPyField` represents handles that are Python object struct fields, i.e.,
+- ``HPyField`` represents handles that are Python object struct fields, i.e.,
   live in native memory attached to some Python object.
 
-- `HPyGlobal` represents handles stored in C global variables. `HPyGlobal`
+- ``HPyGlobal`` represents handles stored in C global variables. ``HPyGlobal``
   can provide isolation between subinterpreters.
 
-**WARNING**: never use a local variable of type ``HPyField``, for any reason!
-If the GC kicks in, it might become invalid and become a dangling pointer.
+.. warning:: Never use a local variable of type ``HPyField``, for any reason! If
+    the GC kicks in, it might become invalid and become a dangling pointer.
 
-**WARNING**: never store `HPy` handles to a long-lived memory, for example: C global
-variables or Python object structs.
+.. warning:: Never store `HPy` handles to a long-lived memory, for example: C
+    global variables or Python object structs.
```

Comment:
Should this be qualified with use of `HPy_Dup`?

Reply:
No, I don't think this is related to `HPy_Dup`. `HPy_Dup` is basically *take ownership* but wouldn't qualify the handle for use in long-lived memory. We can refer to `HPyField_Store` and `HPyGlobal_Store`.

Reply:
ok